xwiki-platform-core/xwiki-platform-filter/xwiki-platform-filter-instance/xwiki-platform-filter-instance-api/src/main/java/org/xwiki/filter/instance/input/InstanceInputProperties.java

```diff
@@ -65,9 +65,7 @@ public void setEntities(EntityReferenceSet entities)
     @Override
     public boolean isVerbose()
     {
-        Boolean verbose = (Boolean) get(PROPNAME_VERBOSE);
-
-        return verbose == Boolean.FALSE ? false : true;
+        return get(PROPNAME_VERBOSE);
```

Comment:
This is not the same thing. If `get(PROPNAME_VERBOSE)` returns null you will end up with a NullPointerException now so this change is wrong.

Reply:
@tmortagne I assumed that the return type of `get(PROPNAME_VERBOSE)` is `boolean` and not `Boolean` as it was cast to `Boolean` a line before. I couldn't find the method `get(..)` so wasn't able to verify. But it can be still be simplified to `return verbose != Boolean.FALSE;` so I changed it back to `return verbose != Boolean.FALSE;` to be on the safer side.

Reply:
@tmortagne I assumed that the return type of `get(PROPNAME_VERBOSE)` is `boolean` and not `Boolean` as it was cast to `Boolean` a line before. I couldn't find the method `get(..)` so wasn't able to verify. But still, it can be simplified to `return verbose != Boolean.FALSE;` so I reverted back to `return verbose != Boolean.FALSE;` to be on the safer side.

Reply:
> get(PROPNAME_VERBOSE) is boolean and not Boolean as it was cast to Boolean a line before

It's cast to `Boolean` because it returns an `Object`.

> so I reverted back to return verbose != Boolean.FALSE;

Yes, that should work fine.